### PR TITLE
Updating custom project to be align with content editing guide

### PR DESCRIPTION
**Pull request by dandara-navarro**

### CODE REVIEW

**Comment by dandara-navarro** [comment]
@mparizeau-pdftron , please review these changes.

File: src/App.js [hunk 1]
```diff
@@ -58,8 +61,33 @@ const App = () => {
...
+    contentEditManager.endContentEditMode();
+    setIsInContentEditMode(false);
+  }
+
+  const addParagraph = () => {
+    if (isInContentEditMode) {
+      const addParagraphTool = documentViewer.getTool(window.Core.Tools.ToolNames.ADD_PARAGRAPH);
+      documentViewer.setToolMode(addParagraphTool);
+    } else {
+      alert('Content Edit mode is not enable.')
```
Comment (mparizeau-pdftron) [inline src/App.js]:
```suggestion
      alert('Content Edit mode is not enabled.')
```

File: src/App.js [hunk 2]
```diff
@@ -58,8 +61,33 @@ const App = () => {
...
+    }
+  };
+
+  const addImageContent = () => {
+    if (isInContentEditMode) {
+      const addImageContentTool = documentViewer.getTool(window.Core.Tools.ToolNames.ADD_IMAGE_CONTENT);
+      documentViewer.setToolMode(addImageContentTool);
+    } else {
+      alert('Content Edit mode is not enable.')
```
Comment (mparizeau-pdftron) [inline src/App.js]:
```suggestion
      alert('Content Edit mode is not enabled.')
```

File: src/App.js [hunk 3]
```diff
@@ -124,13 +152,22 @@ const App = () => {
           <button onClick={editSelectedBox} title="Edit selected box">
             Edit Box
           </button>
+          <button onClick={addParagraph} title="Add new paragraph">
+            <AddParagraph />
+          </button>
+          <button onClick={addImageContent} title="Add new content image">
+            <AddImageContent />
+          </button>
+          <button onClick={endEditingContent} title="End editx mode">
```
Comment (mparizeau-pdftron) [inline src/App.js]:
```suggestion
          <button onClick={endEditingContent} title="End edit mode">
```

File: src/App.js [hunk 4]
```diff
@@ -58,8 +61,33 @@ const App = () => {
...
-    const contentEditTool = documentViewer.getTool(window.Core.Tools.ToolNames.CONTENT_EDIT);
-    documentViewer.setToolMode(contentEditTool);
+    const contentEditManager = documentViewer.getContentEditManager();
+    contentEditManager.startContentEditMode();
+    setIsInContentEditMode(true);
+  }
+
+  const endEditingContent = () => {
+    const contentEditManager = documentViewer.getContentEditManager();
+    contentEditManager.endContentEditMode();
```
Comment (mparizeau-pdftron) [inline src/App.js:67]: I think it would be cleaner too if we also switch to the EDIT tool.

**Review by mparizeau-pdftron — CHANGES_REQUESTED** [review]
1. Now that we have inline editing I think we can remove the modal from this sample and we don't need the "edit box" button anymore.

File: src/App.js [hunk 5]
```diff
@@ -58,8 +61,34 @@ const App = () => {
...
-    const contentEditTool = documentViewer.getTool(window.Core.Tools.ToolNames.CONTENT_EDIT);
-    documentViewer.setToolMode(contentEditTool);
+    const contentEditManager = documentViewer.getContentEditManager();
+    contentEditManager.startContentEditMode();
+    setIsInContentEditMode(true);
+  }
+
+  const endEditingContent = () => {
+    setIsInContentEditMode(false);
+    documentViewer.setToolMode(window.Core.Tools.ToolNames.EDIT)
```
Comment (mparizeau-pdftron) [inline src/App.js]: This should be the following. Right now it's not working.
```suggestion
    documentViewer.setToolMode(documentViewer.getTool(window.Core.Tools.ToolNames.EDIT));
```

**Review by mparizeau-pdftron — CHANGES_REQUESTED** [review]
Since we're not using the modal anymore can you remove all the related code. I still see the `<Modal>` element in the render, we can remove the `simple-react-modal` dependency from package.json, same with all the Quill stuff. All variables and functions related to it can be removed as well.

**Review by mparizeau-pdftron — APPROVED** [review]

**Pull request merged**